NStack/strings/ustring.cs

```diff
@@ -87,27 +88,37 @@ public abstract class ustring : IComparable, IConvertible, IEnumerable<uint>, IE
 
 		// The ustring subclass that supports creating strings for an IntPtr+Size pair.
 		class IntPtrUString : ustring, IDisposable {
```

Comment:
We can keep IDisposable for unit testing purposes, SafeHandle should be able to reclaim whenever it wants.